crypto/bio/bss_file.c

```diff
@@ -89,6 +89,10 @@
 
 # if !defined(OPENSSL_NO_STDIO)
 
+#  if defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_WINDOWS)
+#   include <libc/unconst.h>
+static void dosify_filename(const char *filename);
+#  endif
```

Comment:
Yes, good point all around.  And I like the idea of an auxiliary file, but why uplink?  I'd rather make a new one, say `bio_aux_src`